frontend/src/store/modules/shoots/index.js

```diff
@@ -366,14 +383,23 @@ const mutations = {
   HANDLE_EVENTS (state, { rootState, rootGetters, events }) {
     const onlyShootsWithIssues = get(state, 'shootListFilters.onlyShootsWithIssues', true)
     let setFilteredItemsRequired = false
+    const objectsToAdd = []
+    const shouldAddShoot = (shoot) => {
+      // Do not add healthy shoots when onlyShootsWithIssues=true, this can happen when toggeling flag
+      return rootState.namespace !== '_all' ||
+      !onlyShootsWithIssues ||
+      onlyShootsWithIssues === shootHasIssue(shoot)
+    }
     forEach(events, event => {
       switch (event.type) {
         case 'ADDED':
+          if (shouldAddShoot(event.object)) {
```

Comment:
What is the performance improvement of this change compared to the improvement by replacing the dict of shoots at once. Why do we send events for new shoots if we registered only for shoots without issues? I am not sure if this is needed.